wiiu/vino/vino.ts

```diff
@@ -762,12 +768,12 @@ export default interface VinoAPI {
 	act_getMiiImage: (slot: number) => string;
 
 	/**
-	 * Gets the NNID Mii image for the slot
+	 * Gets the NNID Mii image with a expression for the slot
 	 * @param slot - User account slot. See vino.act_getCurrentSlotNo()
-	 * @param unk2 - Unknown
+	 * @param feeling_id - Feeling ID (2 = HAPPY, 3 = LIKE, etc;)
```

Comment:
We prefer camelCase over snake_case here. Also if there is a range of values, an enum should be used to document them and to restrict inputs to those values. See the `cave` API https://github.com/PretendoNetwork/applet-types/blob/5ab4cb437b9f7cb8840a8a6552ab6fde6082a2bd/3ds/cave/cave.ts#L105-L115